conans/client/rest/file_downloader.py

```diff
@@ -86,24 +95,43 @@ def write_chunks(chunks, path):
                 ret = bytes(ret_data)
             return ret, downloaded_size
 
+        def get_total_length():
+            content_range = response.headers.get("Content-Range", "")
```

Comment:
I am a bit confused about this implementation. If we sent the ``range`` header, then it is mandatory that we get the "Content-Range" response header? So we probably should be looking for the ``Content-Range`` header if and only if we are in ``try_resume=True`` or ``range_start!=0``?